### ISSUE TITLE
About Aaudio err？

### ISSUE THREAD

**Issue by lt1023** [ruffle-rs/ruffle-android#76]
UnsatisfiedLinkError: dlopen failed: library "libaaudio.so" not found

**Comment by torokati44** [comment]
Yes, at the moment, AAudio is required, which was added in Android 8.0 Oreo. There is a way to make audio work on older versions; using OpenSL ES, like how Oboe does it; and it's on the TODO list, but I have no idea when anyone gets to it.

**Comment by lt1023** [comment]
> 是的，目前需要 AAudio，这是在 Android 8.0 Oreo 中添加的。有一种方法可以让音频在旧版本上工作；使用 OpenSL ES，就像 Oboe 的做法一样；它在待办事项列表中，但我不知道什么时候有人会看到它。

well， waiting for your good news ！thks！
